Doc/library/os.rst

```diff
@@ -3951,7 +3951,7 @@ to be ignored.
 
    .. note::
 
-      The standard way to exit is ``sys.exit(n)``.  :func:`_exit` should
+      The standard way to exit is :func:`sys.exit(n) <sys.exit>`.  :func:`_exit` should
```

Comment:
```suggestion
      The standard way to exit is :func:`sys.exit(n) <sys.exit>`.  :func:`!_exit` should
```

While we're at it, we should not cross-reference the function to itself (to avoid confusion).